experimental/Schemes/src/NormalToricVarieties/attributes.jl

```diff
@@ -1,9 +1,8 @@
-# Compute the ideal sheaves and Weil divisors of the rays of `polyhedral_fan(X)` 
-# according to the "orbit-cone-correspondence", Theorem 3.2.6 in 
-# Cox-Little-Schenk.
+# Compute the ideal sheaves and Weil divisors of the rays of `polyhedral_fan(X)`
+# according to the "orbit-cone-correspondence", Theorem 3.2.6 in [CLS11](@cite).
```

Comment:
```suggestion
# according to the "orbit-cone-correspondence", Theorem 3.2.6 in CLS11.
```
the `@cite` is some documenter.jl feature to add links to the bibliography, so it is nonsense to use in comments

Reply:
I agree with @lgoettgens . Also - unless I recall incorrectly (@HechtiDerLachs may correct me on this), we did not anticipate to eventually make those comments part of the documentation. Rather, they are just for internal use to guide developers.

Reply:
Using `[CLS11](@cite)` in comments has the benefit that it is completely unambiguous.
But if writing CLS11 is more common in comments, sure we can do it that way.